UI/window-basic-main.cpp

```diff
@@ -2544,37 +2547,11 @@ void OBSBasic::ShowWhatsNew(const QString &url)
 	if (closing)
 		return;
 
-	std::string info_url = QT_TO_UTF8(url);
-
-	QDialog *dlg = new QDialog(this);
-	dlg->setAttribute(Qt::WA_DeleteOnClose, true);
-	dlg->setWindowTitle("What's New");
-	dlg->resize(700, 600);
-
-	Qt::WindowFlags flags = dlg->windowFlags();
-	Qt::WindowFlags helpFlag = Qt::WindowContextHelpButtonHint;
-	dlg->setWindowFlags(flags & (~helpFlag));
-
-	QCefWidget *cefWidget = cef->create_widget(nullptr, info_url);
-	if (!cefWidget) {
-		return;
+	if (obsWhatsNew) {
+		obsWhatsNew->close();
 	}
 
-	connect(cefWidget, &QCefWidget::titleChanged, dlg, &QDialog::setWindowTitle);
-
-	QPushButton *close = new QPushButton(QTStr("Close"));
-	connect(close, &QAbstractButton::clicked, dlg, &QDialog::accept);
-
-	QHBoxLayout *bottomLayout = new QHBoxLayout();
-	bottomLayout->addStretch();
-	bottomLayout->addWidget(close);
-	bottomLayout->addStretch();
-
-	QVBoxLayout *topLayout = new QVBoxLayout(dlg);
-	topLayout->addWidget(cefWidget);
-	topLayout->addLayout(bottomLayout);
-
-	dlg->show();
+	obsWhatsNew = new OBSWhatsNew(this, QT_TO_UTF8(url));
```

Comment:
If you use `new` in C++ code, you need to balance it with a `delete` somewhere else (which is why in general it's considered bad practice to use `new/delete` in any C++ code outside of library code that manually manages its own resources).

You probably should use a `unique_ptr` (as there is no shared ownership) to ensure the memory is properly released when the pointer object is deallocated (or do the 90s thing and add an explicit `delete` in `OBSBasic`'s destructor).